Revision: devmapper--dilinger--1--patch-6
Archive: dilinger@voxel.net--2004
Creator: Andres Salomon <dilinger@voxel.net>
Date: Mon May 17 03:20:00 EDT 2004
Standard-date: 2004-05-17 07:20:00 GMT
Modified-files: configure.in
New-patches: dilinger@voxel.net--2004/devmapper--dilinger--1--patch-6
Summary: fix libselinux autoconf check
Keywords: 

* added files

    {arch}/devmapper/devmapper--dilinger/devmapper--dilinger--1/dilinger@voxel.net--2004/patch-log/patch-6

* modified files

--- orig/configure.in
+++ mod/configure.in
@@ -152,7 +152,8 @@
 
 dnl Check for is_selinux_enabled
 if test x$SELINUX = xyes; then
-	AC_CHECK_LIB(selinux, is_selinux_enabled, HAVE_SELINUX=yes, HAVE_SELINUX=no)
+	AC_CHECK_HEADERS([selinux/selinux.h],
+		[AC_CHECK_LIB(selinux, is_selinux_enabled, HAVE_SELINUX=yes, HAVE_SELINUX=no)])
 
 if test x$HAVE_SELINUX = xyes; then
 	CFLAGS="$CFLAGS -DHAVE_SELINUX"



